services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -861,35 +870,94 @@ export class NoticeOfWorkApplication extends Component {
             </Tabs.TabPane>
 
             <Tabs.TabPane
-              tab="Referral/Consultation"
-              key="referral-consultation"
+              tab={
+                <span>
```

Comment:
Can we pull this out into a function, as this is being used in the other tabs and they should follow the same format.
```
function renderTabTitle = (title) =>
    (
                <>
                  <NOWStatusIndicator type="badge" />
                  {title}
                </>
    );
```
for example.

Reply:
Yes, I can make this change.

Reply:
Updated!